includes/DataObjects/User.php

```diff
@@ -644,7 +644,7 @@ public function isAdmin()
     
 	public function isCheckuser()
 	{
-		return $this->checkuser == 1 || $this->oauthCanCheckUser();
+		return $this->checkuser == 1 || ($this->isOAuthLinked() && $this->oauthCanCheckUser());
```

Comment:
I thought I'd already caught them all, then this one showed up. Probably worth a hunt around.